vanilla-framework/vanilla-framework
# Update run to 0.3.3

## Done

Update ./run and .gitignore to version 0.3.3 of generator.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Verify still works correctly

Demo should also show up in this PR (fingers crossed)!
